debian/copyright

```diff
@@ -0,0 +1,23 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: neofs-node
+Upstream-Contact: tech@nspcc.ru
```

Comment:
Is this a valid address?

Reply:
Yes, NeoSPCC addresses and urls should be used.

Reply:
We're writing to let you know that the group you tried to contact (tech) may not exist, or you may not have permission to post messages to the group. A few more details on why you weren't able to post:

 * You might have spelled or formatted the group name incorrectly.
 * The owner of the group may have removed this group.
 * You may need to join the group before receiving permission to post.
 * This group may not be open to posting.

If you have questions related to this or any other Google Group, visit the Help Center at https://support.google.com/a/nspcc.ru/bin/topic.py?topic=25838.

Thanks,

nspcc.ru admins


e-mail address must exist

Reply:
Will fix tomorrow

Reply:
still doesn't work